metabase/metabase
# Add basic timeline & timeline events creation

Epic: https://github.com/metabase/metabase/issues/20289
Quick demo: https://www.loom.com/share/0cf6a5271d054683881532b25aa2bb75

Requirements:
- Merge `events-migration` into this branch locally. You might need to resolve a conflict in migration files, accept changes both from the master and `events-migration`

Test steps:
- Go to a collection (e.g. `/collection/root`)
- Click on the calendar icon in the collection header
- You should see an empty state with a link to add your first event, click it
- Fill in the name and the date (text only for now) for the event, click `Create`
- You should be navigated to a newly created timeline. Please note that events are currently not returned by the backend
- Click `Add an event`. It should be possible to add another event
- Go to a collection (e.g. `/collection/root`) again and click on the calendar icon
- You should be able to see a list with your timeline
- It should be possible to create another timeline from the modal menu

Known issues:
- It's a simplified version of what's in designs, I'll improve it in follow-up PRs
